controllers/routes.js

```diff
@@ -227,9 +227,12 @@ sections.about.addRoutes({
             '/england/about-big',
             '/wales/about-big',
             '/scotland/about-big',
-            '/northernireland/about-big',
+            '/northernireland/about-big'
         ]
     }),
+    content: cmsRoute({
+        path: '/*'
```

Comment:
This seems a bit too general, but does this just mean that any page under `/about/` will first be tried against the CMS, then against the legacy site, then as a 404?

Reply:
Yep, exactly.

Reply:
Carry on!